package.json

```diff
@@ -274,7 +274,7 @@
     "@metamask/address-book-controller": "^7.0.0",
     "@metamask/announcement-controller": "^8.0.0",
     "@metamask/approval-controller": "^8.0.0",
-    "@metamask/assets-controllers": "^89.0.1",
+    "@metamask/assets-controllers": "patch:@metamask/assets-controllers@npm%3A89.0.1#~/.yarn/patches/@metamask-assets-controllers-npm-89.0.1-02fa7acd54.patch",
```

Comment:
All changes in this file come from this PR https://github.com/MetaMask/metamask-extension/pull/38126, which was not included in 13.11.0, but was cherry-picked in 13.10.1.